src/peergos/shared/user/fs/FileWrapper.java

```diff
@@ -444,6 +444,12 @@ public SymmetricKey getParentKey() {
                 .thenApply(children -> children.stream().filter(f -> f.getName().equals(name)).findAny());
     }
 
+    public CompletableFuture<Optional<FileWrapper>> getUpdatedChild(String name, Hasher hasher, NetworkAccess network) {
```

Comment:
Can you explain why we need this?

Reply:
Was part of optimistic locking check. No longer required

Reply:
Can we remove it if it's no longer needed?